models/workitem_repository.go

```diff
@@ -132,6 +134,22 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, typeID string, fiel
 		Type:   typeID,
 		Fields: Fields{},
 	}
+	publicKey, err := token.ParsePublicKey(token.RSAPublicKey)
```

Comment:
This can't go here. Most of this logic needs to move to the /workitem.go REST endpoint.